src/reactor.h

```diff
@@ -0,0 +1,342 @@
+    "uilabel": "Blanket Turnover Rate" \
+  }
+  double blanket_turnover_rate;
+
+  #pragma cyclus var { \
+    "default": 1, \
+    "doc": "number of timesteps between blanket recycles", \
+    "tooltip": "Defaults to 0.05 (5%), must be between 0 and 15%", \
+    "units": "dimensionless", \
+    "uitype": "range", \
+    "range": [0, 1000], \
+    "uilabel": "Blanket Turnover Rate" \
+  }
+  int blanket_turnover_frequency;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling tritium material to be used in reactor"}
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_storage;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling excess tritium material to be sold"}
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_excess;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling tritium sequestered in the system"}
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_sequestered;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling helium-3 byproduct material"}
+  cyclus::toolkit::ResBuf<cyclus::Material> helium_storage;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling enriched lithium blanket material"}
+  cyclus::toolkit::ResBuf<cyclus::Material> blanket;
+
+  #pragma cyclus var {"tooltip":"Tracker to handle on-hand tritium"}
+  cyclus::toolkit::TotalInvTracker fuel_tracker;
+
+  #pragma cyclus var {"tooltip":"Tracker to handle blanket material"}
+  cyclus::toolkit::TotalInvTracker blanket_tracker;
+
+  cyclus::toolkit::MatlBuyPolicy fuel_startup_policy;
+  cyclus::toolkit::MatlBuyPolicy fuel_refill_policy;
+
+  cyclus::toolkit::MatlBuyPolicy blanket_fill_policy;
+
+  cyclus::toolkit::MatlSellPolicy tritium_sell_policy;
+  cyclus::toolkit::MatlSellPolicy helium_sell_policy;
+
+  void Startup();
+  void OperateReactor(double TBR);
+  void SequesterTritium();
+  void DecayInventory(cyclus::toolkit::ResBuf<cyclus::Material> &inventory);
+  void CombineInventory(cyclus::toolkit::ResBuf<cyclus::Material> &inventory);
+  void ExtractHelium(cyclus::toolkit::ResBuf<cyclus::Material> &inventory);
+  void RecordEvent(std::string name, std::string val);
+  void RecordStatus(std::string Status, double power);
+  void RecordInventories(double storage, double excess, double sequestered, double blanket, double helium);
+  void RecordOperationalInfo(std::string name, std::string val);
+  void DepleteBlanket(double bred_tritium_mass);
+  cyclus::Material::Ptr BreedTritium(double fuel_usage, double TBR);
+  std::string GetComp(cyclus::Material::Ptr mat);
+
+
+  private: 
+    bool sufficient_tritium_for_operation = false;
+    const double seconds_per_year = 31536000.0;
+    const double MW_to_GW = 1000.0;
+    double fuel_usage;
+    std::string expected_fuel_comp = "{{10030000,1.000000}}";
+
+    //masses in kg/atom
+    const double Li7_atomic_mass = 1.16503478e-26;
+    const double Li6_atomic_mass = 9.9883465e-27;
+    const double tritium_atomic_mass = 5.00826766e-27;
+    const double He4_atomic_mass = 6.64647907e-27;
```

Comment:
See `pyne::atomic_mass()`

Reply:
Yep, didn't know that was a thing. Good suggestion!